Eventinator/Eventinator/Models/Category.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Category.swift
+//  Lineup
+//
+//  Created by Evelio Tarazona on 11/12/16.
+//  Copyright © 2016 Eventinator. All rights reserved.
+//
+
+import Foundation
+
+struct Category {
```

Comment:
Do you need to keep these model classes/structs in separate .swift files? I would prefer to keep them in the same files as much as possible, especially for classes/structs with 1-1 or 1-many relationships.